src/core/quantum_gate.jl

```diff
@@ -322,6 +434,9 @@ function apply_gate!(state::Ket, gate::AbstractGate)
     apply_operator!(state,operator,connected_qubits)
 end
 
+# TODO(#226): delete on completion
+apply_gate!(state::Ket, gate::Gate) = apply_gate!(state::Ket, get_gate(gate))
```

Comment:
It seems to me this is the signature we should keep

Reply:
See above.